miniapps/nurbs/nurbs_stokes.cpp

```diff
@@ -0,0 +1,793 @@
+   args.AddOption(&schur_maxiter, "-sli", "--schur_linear-itermax",
+                  "Maximum iteration count for the Schur complement GMRES solver.");
+
+#ifdef MFEM_USE_SUITESPARSE
+   bool UMFPack = true;
+   args.AddOption(&UMFPack, "-umf", "--direct", "-gs", "--gssmoother",
+                  "Type of preconditioner to use.");
+#endif
+
+   args.Parse();
+   if (!args.Good())
+   {
+      args.PrintUsage(cout);
+      return 1;
+   }
+   args.PrintOptions(cout);
+
+   if (schur_complement)
+   {
+#ifdef MFEM_USE_SUITESPARSE
+      if (UMFPack)
+      {
+         std::cout<<"Schur complement is exact"<<std::endl;
+      }
+      else
+      {
+         std::cout<<"Schur complement is approximate!!"<<std::endl;
+      }
+#else
+      std::cout<<"Schur complement is approximate"<<std::endl;
+      std::cout<<" - Compile with SuiteSparse to have an exact Schur complement"<<std::endl;
+#endif
+   }
+
+   if (penalty < 0)
+   {
+      penalty = 10*(order+1)*(order+1);
+   }
+
+   // Enable hardware devices such as GPUs, and programming models such as
+   // CUDA, OCCA, RAJA and OpenMP based on command line options.
+   Device device(device_config);
+   device.Print();
+
+   // Read the mesh from the given mesh file.
+   Mesh *mesh = new Mesh(mesh_file, 1, 1);
+   int dim = mesh->Dimension();
+
+   // Refine the mesh to increase the resolution.
+   {
+      if (ref_levels < 0)
+      {
+         ref_levels =
+            (int)floor(log(10000./mesh->GetNE())/log(2.)/dim);
+      }
+      for (int l = 0; l < ref_levels; l++)
+      {
+         mesh->UniformRefinement();
+      }
+   }
+
+   // Define a finite element space on the mesh.
+   FiniteElementCollection *hdiv_coll = nullptr;
+   FiniteElementCollection *l2_coll = nullptr;
+   NURBSExtension *NURBSext = nullptr;
+
+   if (mesh->NURBSext && !pa)
+   {
+      if (boundary_conditions == BC::STRONG)
+      {
+         hdiv_coll = new NURBS_HDivH1FECollection(order,dim);
+      }
+      else
+      {
+         hdiv_coll = new NURBS_HDivFECollection(order,dim);
+      }
+      l2_coll   = new NURBSFECollection(order);
+      NURBSext  = new NURBSExtension(mesh->NURBSext, order);
+      mfem::out<<"Create NURBS fec and ext"<<std::endl;
+   }
+   else
+   {
+      hdiv_coll = new RT_FECollection(order, dim);
+      l2_coll   = new L2_FECollection(order, dim);
+      mfem::out<<"Create Normal fec"<<std::endl;
+   }
+
+   FiniteElementSpace p_space(mesh, NURBSext, l2_coll);
+   FiniteElementSpace u_space(mesh,
+                              p_space.StealNURBSext(),
+                              hdiv_coll);
+
+   Array<int> ess_bdr(mesh->bdr_attributes.Max());
+   ess_bdr = 1;
+
+   // Define the BlockStructure of the problem
+   Array<int> bOffsets(3); // number of variables + 1
+   bOffsets[0] = 0;
+   bOffsets[1] = u_space.GetVSize();
+   bOffsets[2] = p_space.GetVSize();
+   bOffsets.PartialSum();
+
+   std::cout<<"===========================================================\n";
```

Comment:
`std::cout` -> `cout`

Reply:
Dropped the `std::`  :)

Also in the vector-diffusion case.